test/performance/alignment/edit_distance_unbanded_benchmark.cpp

```diff
@@ -27,11 +27,10 @@
 #include <seqan/sequence.h>
 #endif
 
-using namespace seqan3;
 using namespace seqan3::test;
```

Comment:
Do we keep using namespace seqan3::test but not namespace seqan3 ?
Should we not be consistent and also replace the "using namespace seqan3::test" ?

Reply:
Good question :)
We removed the `using seqan3` because we want to test exactly how the user is using our library. There is e a difference for the compiler between
`to_char(...)` -> **unqualified** lookup, and
`seqan3::to_char(...)` **qualified** lookup.
By not testing the qualified lookup that we recommend to the user we might not catch subtle bugs.
In contrast, `seqan3::test` is a pure internal namespace, never to be used by anyone other than us, so there is no need to explicitly test the qualified lookup.

Reply:
I understand. Thanks for the explanation !:)